client/src/Faq/Faq.scss

```diff
@@ -0,0 +1,17 @@
+@import "../common_css/common-variables";
+
+ul.fancy-ul.faq-index {
+  & > li {
+    border-color: $primaryColor;
+    padding: 0px 15px;
+
+    &:first-child {
```

Comment:
I think these could be generalized into fancyul scss for potential future use. This is basically fancy ul with title

Reply:
Oh, hmm, didn't notice because the CSS is buried in `site.scss`, but there's already a class for `fancy-ul-title-row`. The `FancyUL` styling's a bit of a mess, I think it was pre-react, so the react component for it is pretty slapped together/the styles aren't co-located with it.

I think the `FancyUL` component should be moved from `misc_util_components` to its own file, its styling should be plucked from `site.scss`, and then the whole thing should be refactored a little. Can make a consistent way of adding a title to it at that point.

Bit out of scope for this PR, but I might slip it in since I pretty much thought through how to do that in the process of writing this comment, haha. Don't really want to open more issues right now anyway.